### ISSUE TITLE
Add items tree sorting options to View menu

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2273]
In https://github.com/zotero/zotero/pull/2263#pullrequestreview-824718234, I brought up accessing the column headers via the keyboard, but that's not the way to do it, of course — it should be in the View menu.

E.g., in Finder, there's a View → Sort By menu that's equivalent to clicking on a column header, and it also has numbered key bindings for each column (e.g., Cmd-Option-Ctrl-1, though weirdly I can't find a way to sort descending via the keyboard — seems like at least triggering the same column twice should toggle the direction.

And then, separately, Finder has a Show View Options option that opens a dialog to hide/show panes, though that's in part to enable folder-specific settings. We could do it in a separate menu that just mirrored the context menu.

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/99836/mac-sort-articles-by-the-year-published

We've been getting a ton of these since Zotero 6 (even though we're now using the OS-standard approach in the column header).
